server.js

```diff
@@ -0,0 +1,75 @@
+const express = require("express");
+const server = express();
+const bodyParser = require("body-parser")
+
+let mysql = require('mysql')
+
+let connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'root',
+    password: '',
+    database: 'users'
+})
+
+connection.connect(function(err){
+    if(err) throw err
+        console.log("You're connected.")  
+})
+
+const urlencodedParser = bodyParser.urlencoded({extended: false});
+
+server.set("view-engine", "ejs")
+//server.use(express.urlencoded({ extended: false}));
+
+server.get("/", (req, res)=>{
+    res.render("index.ejs")
+})
+
+server.post("/", urlencodedParser, (req, res)=>{
```

Comment:
please use `urlencodedParser` globally and not at each route. this is very tedious.

Reply:
okay. thanks for the feedback